Bio/Graphics/KGML_vis.py

```diff
@@ -32,7 +32,7 @@
 
     raise MissingPythonDependencyError("Install pillow if you want to use KGML_vis.")
 
-from Bio._py3k import urlopen as _urlopen
+from urllib.request import urlopen as _urlopen
```

Comment:
underscores can be dropped in this files and all others

Reply:
Not relevant to the py3k removal, but perhaps worth discussing on a separate issue?

Reply:
I am fine with a separate issue for resolving underscore removal.

Reply:
Let's discuss this on #2483

Reply:
Let's discuss this on #2483